src/bacnet/basic/object/ai.c

```diff
@@ -70,6 +78,9 @@ static const int Properties_Optional[] = { PROP_DESCRIPTION, PROP_RELIABILITY,
 
 static const int Properties_Proprietary[] = { 9997, 9998, 9999, -1 };
 
+ANALOG_INPUT_DESCR * Analog_Input_Allocate();
```

Comment:
@Michail-Antropov Implement dynamic object handling in an independent ticket/PR from OBJECT_TABLE.